hotel/models/hotel.py

```diff
@@ -0,0 +1,136 @@
+# -*- encoding: utf-8 -*-
+from openerp import models
+from openerp import fields
+
+
+class hotel_floor(models.Model):
```

Comment:
Camelcase needs to be used for V8 class [OCA guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#symbols)